### PR TITLE
ci: add react-18 to tests

### PR DESCRIPTION
**Pull request by Yeti-or**
Реакт полностью переключен на 18 версию
Storybook теперь на react 18 
Cypress + jest гоняются на двух версиях: 18 & 17

--
отключен стаб для [Editable:selectText](https://github.com/salute-developers/plasma/issues/384)

:baby_chick: Download canary assets:

[colors--canary.194.4342208887.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.194.4342208887.xml)  
[color_metro_ios-swift--canary.194.4342208887.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_ios-swift--canary.194.4342208887.swift)  
[color_metro_kotlin--canary.194.4342208887.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_kotlin--canary.194.4342208887.kt)  
[color_metro_react-native--canary.194.4342208887.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_react-native--canary.194.4342208887.ts)  
[color_sberHealth_ios-swift--canary.194.4342208887.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.194.4342208887.swift)  
[color_sberHealth_kotlin--canary.194.4342208887.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.194.4342208887.kt)  
[color_sberHealth_react-native--canary.194.4342208887.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.194.4342208887.ts)  
[color_sbermarket_ios-swift--canary.194.4342208887.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.194.4342208887.swift)  
[color_sbermarket_kotlin--canary.194.4342208887.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.194.4342208887.kt)  
[color_sbermarket_react-native--canary.194.4342208887.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.194.4342208887.ts)  
[color_sberprime_ios-swift--canary.194.4342208887.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.194.4342208887.swift)  
[color_sberprime_kotlin--canary.194.4342208887.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.194.4342208887.kt)  
[color_sberprime_react-native--canary.194.4342208887.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.194.4342208887.ts)  
[color_selgros_ios-swift--canary.194.4342208887.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_ios-swift--canary.194.4342208887.swift)  
[color_selgros_kotlin--canary.194.4342208887.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_kotlin--canary.194.4342208887.kt)  
[color_selgros_react-native--canary.194.4342208887.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_react-native--canary.194.4342208887.ts)  
[color_smbusiness_ios-swift--canary.194.4342208887.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_ios-swift--canary.194.4342208887.swift)  
[color_smbusiness_kotlin--canary.194.4342208887.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_kotlin--canary.194.4342208887.kt)  
[color_smbusiness_react-native--canary.194.4342208887.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_react-native--canary.194.4342208887.ts)  
[PlasmaTokensColor--canary.194.4342208887.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.194.4342208887.swift)  
[shadow_sbermarket_react-native--canary.194.4342208887.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.194.4342208887.ts)  
[typo_mage_ios-swift--canary.194.4342208887.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.194.4342208887.swift)  
[typo_mage_kotlin--canary.194.4342208887.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.194.4342208887.kt)  
[typo_mage_react-native--canary.194.4342208887.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.194.4342208887.ts)  
[typo_plasma_ios-swift--canary.194.4342208887.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.194.4342208887.swift)  
[typo_plasma_kotlin--canary.194.4342208887.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.194.4342208887.kt)  
[typo_plasma_react-native--canary.194.4342208887.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.194.4342208887.ts)  
[typo_ruler_ios-swift--canary.194.4342208887.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.194.4342208887.swift)  
[typo_ruler_kotlin--canary.194.4342208887.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.194.4342208887.kt)  
[typo_ruler_react-native--canary.194.4342208887.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.194.4342208887.ts)  
[typo_sage_ios-swift--canary.194.4342208887.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.194.4342208887.swift)  
[typo_sage_kotlin--canary.194.4342208887.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.194.4342208887.kt)  
[typo_sage_react-native--canary.194.4342208887.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.194.4342208887.ts)  
[typo_sbermarket_ios-swift--canary.194.4342208887.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.194.4342208887.swift)  
[typo_sbermarket_kotlin--canary.194.4342208887.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.194.4342208887.kt)  
[typo_sbermarket_react-native--canary.194.4342208887.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.194.4342208887.ts)  
[typo_soulmate_ios-swift--canary.194.4342208887.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.194.4342208887.swift)  
[typo_soulmate_kotlin--canary.194.4342208887.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.194.4342208887.kt)  
[typo_soulmate_react-native--canary.194.4342208887.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.194.4342208887.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.152.2-canary.194.4342208887.0
  npm install @salutejs/plasma-core@1.100.2-canary.194.4342208887.0
  npm install @salutejs/plasma-hope@0.11.2-canary.194.4342208887.0
  npm install @salutejs/plasma-icons@1.128.2-canary.194.4342208887.0
  npm install @salutejs/plasma-temple@1.139.2-canary.194.4342208887.0
  npm install @salutejs/plasma-tokens@1.38.1-canary.194.4342208887.0
  npm install @salutejs/plasma-typo@0.31.1-canary.194.4342208887.0
  npm install @salutejs/plasma-ui@1.172.2-canary.194.4342208887.0
  npm install @salutejs/plasma-web@1.177.2-canary.194.4342208887.0
  npm install @salutejs/plasma-cy-utils@0.45.1-canary.194.4342208887.0
  npm install @salutejs/plasma-sb-utils@0.98.2-canary.194.4342208887.0
  npm install @salutejs/plasma-tokens-android@2.45.1-canary.194.4342208887.0
  npm install @salutejs/plasma-tokens-ios-swift@2.45.1-canary.194.4342208887.0
  # or 
  yarn add @salutejs/plasma-b2c@1.152.2-canary.194.4342208887.0
  yarn add @salutejs/plasma-core@1.100.2-canary.194.4342208887.0
  yarn add @salutejs/plasma-hope@0.11.2-canary.194.4342208887.0
  yarn add @salutejs/plasma-icons@1.128.2-canary.194.4342208887.0
  yarn add @salutejs/plasma-temple@1.139.2-canary.194.4342208887.0
  yarn add @salutejs/plasma-tokens@1.38.1-canary.194.4342208887.0
  yarn add @salutejs/plasma-typo@0.31.1-canary.194.4342208887.0
  yarn add @salutejs/plasma-ui@1.172.2-canary.194.4342208887.0
  yarn add @salutejs/plasma-web@1.177.2-canary.194.4342208887.0
  yarn add @salutejs/plasma-cy-utils@0.45.1-canary.194.4342208887.0
  yarn add @salutejs/plasma-sb-utils@0.98.2-canary.194.4342208887.0
  yarn add @salutejs/plasma-tokens-android@2.45.1-canary.194.4342208887.0
  yarn add @salutejs/plasma-tokens-ios-swift@2.45.1-canary.194.4342208887.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-194/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-194/
  ui storybook: http://plasma.sberdevices.ru/pr-194/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-194/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-194/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-194/temple-storybook/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-194/
  ui storybook: http://plasma.sberdevices.ru/pr-194/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-194/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-194/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-194/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🌚 Missing base report**

Check out current report in job artifacts!

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-194/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🌚 Missing base report**

Check out current report in job artifacts!

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-194/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-194/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🌚 Missing base report**

Check out current report in job artifacts!

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-194/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **💀 Missing report, something went wrong**

Check out current report in job artifacts!

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-194/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **💀 Missing report, something went wrong**

Check out current report in job artifacts!

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-194/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **💀 Missing report, something went wrong**

Check out current report in job artifacts!

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **💀 Missing report, something went wrong**

Check out current report in job artifacts!

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-194/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **💀 Missing report, something went wrong**

Check out current report in job artifacts!

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **💀 Missing report, something went wrong**

Check out current report in job artifacts!

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **💀 Missing report, something went wrong**

Check out current report in job artifacts!

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-194/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-194/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🌚 Missing base report**

Check out current report in job artifacts!

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-194/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **💀 Missing report, something went wrong**

Check out current report in job artifacts!

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-194/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-194/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-194/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-194/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-194/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-194/temple-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-194/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **💀 Missing report, something went wrong**

Check out current report in job artifacts!

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-194/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-194/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-194/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-194/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-194/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-194/temple-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-194/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **💀 Missing report, something went wrong**

Check out current report in job artifacts!

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-194/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-194/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-194/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-194/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-194/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-194/temple-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-194/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **💀 Missing report, something went wrong**

Check out current report in job artifacts!

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-194/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-194/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-194/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-194/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-194/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-194/temple-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-194/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **💀 Missing report, something went wrong**

Check out current report in job artifacts!

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-194/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-194/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-194/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-194/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-194/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-194/temple-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-194/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **💀 Missing report, something went wrong**

Check out current report in job artifacts!

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-194/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-194/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-194/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-194/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-194/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-194/temple-storybook/

File: packages/plasma-hope/src/components/Editable/Editable.component-test.tsx [hunk 1]
```diff
@@ -75,8 +75,9 @@ describe('plasma-hope: Editable', () => {
 
         // для случаев, если не поддерживаются современные интерфейсы window
         cy.window().then((win) => {
+            // NOTE: https://github.com/salute-developers/plasma/issues/384
```
Comment (Yeti-or) [inline packages/plasma-hope/src/components/Editable/Editable.component-test.tsx:78]: https://github.com/salute-developers/plasma/issues/384

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-194/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **💀 Missing report, something went wrong**

Check out current report in job artifacts!

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-194/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-194/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-194/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-194/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-194/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-194/temple-storybook/

**Review by Yakutoc — APPROVED** [review]

**Comment by neretin-trike** [comment]
добавишь небольшое описание? Про то что, тесты добавил, версию реакта переключил и т.д.

File: .github/workflows/cypress-ct-17.yml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        key: npm-deps-${{ hashFiles('package-lock.json') }}
+        restore-keys: |
+          npm-deps-${{ hashFiles('package-lock.json') }}
+
+    - name: Setup packages
+      run: |
+        npm i
+        npm i react@17 react-dom@17 --prefix="./packages/plasma-ui"
+        npm i react@17 react-dom@17 --prefix="./packages/plasma-temple"
+        npm i react@17 react-dom@17 --prefix="./packages/plasma-hope"
```
Comment (neretin-trike) [inline .github/workflows/cypress-ct-17.yml:32]: такс, а тут не надо ещё  `npm i react@17 react-dom@17 --prefix="./packages/plasma-core"`? Там же тоже реакт имеется, и зависимые компоненты

File: .github/workflows/tests-17.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+          key: npm-deps-${{ hashFiles('package-lock.json') }}
+          restore-keys: |
+            npm-deps-${{ hashFiles('package-lock.json') }}
+
+      - name: Setup packages
+        run: |
+          npm i
+          npm i react@17 react-dom@17 --prefix="./packages/plasma-ui"
+          npm i react@17 react-dom@17 --prefix="./packages/plasma-temple"
+          npm i react@17 react-dom@17 --prefix="./packages/plasma-hope"
```
Comment (neretin-trike) [inline .github/workflows/tests-17.yml:42]: а тут нужен plasma-core?

File: packages/plasma-core/api/plasma-core.api.md [hunk 4]
```diff
@@ -1106,7 +1106,7 @@ export const tertiary = "var(--plasma-colors-tertiary)";
 export const text = "var(--plasma-colors-text)";
 
 // @public
-const TextArea: StyledComponent<"textarea", any, Pick<TextAreaProps, "resize" | "status">, never>;
+const TextArea: StyledComponent<"textarea", any, Pick<TextAreaProps, "status" | "resize">, never>;
```
Comment (neretin-trike) [inline packages/plasma-core/api/plasma-core.api.md:1109]: :D

Comment (Yeti-or) [reply packages/plasma-core/api/plasma-core.api.md:1109]: Оно само ))

File: .github/workflows/cypress-ct-17.yml (hunk 2 above)
Comment (Yeti-or) [reply .github/workflows/cypress-ct-17.yml:32]: я решил что можно оставить только конечные пакеты чтобы оно быстрее ставилось, в плане реакт же резолвится там где испоняются тесты, думаю что кстати plasm-hope можно тоже удалить тут

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-194/

**Review by neretin-trike — APPROVED** [review]

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **💀 Missing report, something went wrong**

Check out current report in job artifacts!

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-194/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-194/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-194/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-194/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-194/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-194/temple-storybook/

**Pull request merged**